airscan-id.c

```diff
@@ -177,6 +177,40 @@ id_format_short_name (ID_FORMAT id)
     return name ? name : mime;
 }
 
+/******************** ID_SCANINTENT ********************/
+/* id_scanintent_sane_name_table represents ID_SCANINTENT to
+ * SANE name mapping
+ */
+static id_name_table id_scanintent_sane_name_table[] = {
+    {ID_SCANINTENT_AUTO,           "Auto"},
+    {ID_SCANINTENT_DOCUMENT,       "Text"},
+    {ID_SCANINTENT_TEXTANDGRAPHIC, "Mixed"},
```

Comment:
Better "TextAndGraphic" instead of "Mixed".

Reply:
Changed it to "Text and Graphic".